src/Akka.TestKit.NUnit3/Akka.TestKit.NUnit3.csproj

```diff
@@ -2,13 +2,13 @@
 
   <PropertyGroup>
     <AssemblyTitle>Akka.TestKit.NUnit</AssemblyTitle>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>

```

Comment:
Does NUnit3 no longer support .NET Standard 2.0?

Reply:
Hmm I feel like there's a reason I did this but the context has left me since.

Give me some time tonight and I'll double check and be sure.